Data-Structure-Stacks.md

```diff
@@ -0,0 +1,63 @@
+# Data Structure Stacks
+
+`Stack` is a linear data structure which follows a particular order in which the operations are performed. The order may be 
+LIFO(Last In First Out) or FILO(First In Last Out).
+
+The basic three operations which can be performed on a `Stack`
+
+1. `Push`: Adds an item in the `Stack`. If the stack is full, then it is said to be an Overflow condition. 
+2. `Pop`: Removes an item from the `Stack`. The items are popped in the reversed order in which they are pushed. If the `Stack` is empty, 
+           then it is said to be an Underflow condition. 
+3. `Peek`: Gets the topmost item of the `Stack`. It does not removes the item from the stack.
+
+## Real life example of a `Stack`
+
+Consider a number of plates kept one upon the one another on a dining table. Now the first plate which we have kept would be the last plate
```

Comment:
This doesn't make much sense. It could be re-factored. I'm not sure what you mean, so can't give guidance.